tuftool/README.md

```diff
@@ -115,8 +115,29 @@ tuftool update \
    --timestamp-version 2 \
    --outdir "${WRK}/tuf-repo" \
    --metadata-url file:///$WRK/tuf-repo/metadata
+
+#[Optional] Set an RHTAS target (fulcio, ctlog, rekor, tsa)!
+touch "${WRK}/input/ctfe.pub" 
+
+tuftool rhtas \
+   --root "${ROOT}" \
+   --key "${WRK}/keys/root.pem" \
+   --set-ctlog-target "${WRK}/input/ctfe.pub" \
+   --ctlog-uri "https://ctfe.sigstore.dev" \
+   --ctlog-usage "CTFE" \
```

Comment:
If the user is adding a CTLog target, they shouldn't need to specify usage, because the usage is always "CTFE", isn't it? (same for all other targets, adding a fulcio cert should automatically set usage to "Fulcio" etc)